ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -38,14 +38,16 @@ Loader {
     property string messageId: ""
     property string communityId: ""
     property string responseToMessageWithId: ""
+    property var responseToMessage: getReplyMessage()
 
     property string senderId: ""
     property string senderDisplayName: ""
     property string senderOptionalName: ""
     property bool senderIsEnsVerified: false
     property string senderIcon: ""
+    property var senderColorHash:  senderId != "" ? Utils.getColorHashAsJson(senderId, senderIsEnsVerified) : ""
```

Comment:
Don't we get this from the model already?

Reply:
True. Must have missed it on merge. Will update

Reply:
Actually there is a reason why I kept the default to Utils.getColorHashAsJson(senderId, senderIsEnsVerified). It's because the MessageView is used in different scenarios, like ContactVerificationRequest, OngoingContactVerificationRequest or PinnedMessagesPopup.

Now there are two options: Leave the default like this or update all models.
Performance wise it has no impact on the delegate because the senderColorHash is provided from above.

Reply:
Ah I see... the value is not always coming from the model :/ I'd say keep it for now, add a FIXME/TODO for later